<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenMainArtica</class>
 <widget class="QWidget" name="screenMainArtica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="button_cambiaLogin">
         <property name="text">
          <string>Cambia utente</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_utenteLog">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="w_toolBar" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="layoyt_toolbar"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filtri</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Descrizione:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_cercaDescrizioneArticolo">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Tipo:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboCercaTipo">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="l_espansione_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Espansione: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboCercaEspansione">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="l_espansione">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_cercaCodiceArticolo">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Codice articolo:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="tb_attivaCercaArticolo">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="tb_cerca">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>601</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Panoramica</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QTableView" name="tableViewPanoramica"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Dettagli</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_21">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Codice articolo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="l_InsertCodiceArticolo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Descrizione:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="l_InsertDescrizioneArticolo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Espansione: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboInsertEspansione"/>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="l_insertEspansione">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="tb_modEspansione">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_17">
         <item row="0" column="2">
          <widget class="QLineEdit" name="l_insertTipo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Tipo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboInsertTipo"/>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="tb_modTipo">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Rarita':</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="l_InsertRarezza">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Lingua:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="l_InsertLingua">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>Data:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="l_InsertData">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_19">
           <item row="0" column="0">
            <widget class="QCalendarWidget" name="calendarWidgetDataIns"/>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <spacer name="horizontalSpacer_15">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>589</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>522</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_cambiaLogin</tabstop>
  <tabstop>tb_cerca</tabstop>
  <tabstop>l_cercaCodiceArticolo</tabstop>
  <tabstop>tb_attivaCercaArticolo</tabstop>
  <tabstop>l_cercaDescrizioneArticolo</tabstop>
  <tabstop>comboCercaEspansione</tabstop>
  <tabstop>l_espansione</tabstop>
  <tabstop>comboCercaTipo</tabstop>
  <tabstop>l_espansione_2</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableViewPanoramica</tabstop>
  <tabstop>l_InsertCodiceArticolo</tabstop>
  <tabstop>l_InsertDescrizioneArticolo</tabstop>
  <tabstop>comboInsertEspansione</tabstop>
  <tabstop>l_insertEspansione</tabstop>
  <tabstop>tb_modEspansione</tabstop>
  <tabstop>comboInsertTipo</tabstop>
  <tabstop>l_insertTipo</tabstop>
  <tabstop>tb_modTipo</tabstop>
  <tabstop>l_InsertRarezza</tabstop>
  <tabstop>l_InsertLingua</tabstop>
  <tabstop>l_InsertData</tabstop>
  <tabstop>calendarWidgetDataIns</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
